trie/trieStorageManager.go

```diff
@@ -613,10 +635,14 @@ func (tsm *trieStorageManager) SetEpochForPutOperation(epoch uint32) {
 	storer.SetEpochForPutOperation(epoch)
 }
 
+// EpochConfirmed is called whenever a new epoch is confirmed
 func (tsm *trieStorageManager) EpochConfirmed(epoch uint32, _ uint64) {
 	tsm.flagDisableOldStorage.Toggle(epoch >= tsm.disableOldStorageEpoch)
 	log.Debug("old trie storage", "disabled", tsm.flagDisableOldStorage.IsSet())
 
+	err := tsm.mainStorer.Put([]byte(activeDBKey), []byte(activeDBVal))
```

Comment:
it looks like this is the only place when you set the value for active DB. I guess it is meant to be set only on the first confirmed epoch, right? So if starting from genesis, epoch 0 -> epoch 1 will trigger the value to be set; or when starting in epoch, only the next epoch?

Reply:
This func is called every time when an epoch is changed, or when the trieStorageManager is initialized, so `activeDBKey` will be saved in each storer (each epoch a new storer is created)